x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/Connector.java

```diff
@@ -447,8 +447,28 @@ public String getConnectorId() {
         return connectorId;
     }
 
-    public ConnectorScheduling getScheduling() {
-        return scheduling;
+    public String getApiKeyId() {
```

Comment:
Should this be hashed or something if we return it via an API call?

Reply:
This is just and id of api key, it's ok to return it via api call